app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+
+  get '/login' do
+    if !!session[:id]
```

Comment:
You could use the `current_user` and `logged_in?` helper methods here, which is preferable to referring to the session object directly.